app/livechat/server/methods/saveAppearance.js

```diff
@@ -12,6 +12,7 @@ Meteor.methods({
 		const validSettings = [
 			'Livechat_title',
 			'Livechat_title_color',
+			'Livechat_title_character_limit',
```

Comment:
Im having trouble getting this to save correctly. When I tried debugging the setting below...
```
		settings.forEach((setting) => {
			rcSettings.updateById(setting._id, setting.value);
                        console.log(`========DEBUG ${ setting._id } ${ setting.value }`.green)
                        console.log(rcSettings.get(setting._id))
		});
```
i got
```
I20200722-11:58:45.627(-4)? ========DEBUG Livechat_title Rocket.Chat
I20200722-11:58:45.627(-4)? Rocket.Chat
I20200722-11:58:45.632(-4)? ========DEBUG Livechat_title_color #366bc4
I20200722-11:58:45.632(-4)? #366bc4
I20200722-11:58:45.635(-4)? ========DEBUG Livechat_title_character_limit 13
I20200722-11:58:45.636(-4)? undefined
I20200722-11:58:45.638(-4)? ========DEBUG Livechat_show_agent_info true
I20200722-11:58:45.638(-4)? true
I20200722-11:58:45.640(-4)? ========DEBUG Livechat_show_agent_email true
I20200722-11:58:45.640(-4)? true
I20200722-11:58:45.642(-4)? ========DEBUG Livechat_display_offline_form false
I20200722-11:58:45.642(-4)? false
```